README.md

```diff
@@ -1 +1,72 @@
-# capstone-js-movies
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone JS Movie app
+
+> Microverse project on practcing gitflow, webpack, Javascript, consuming API's,
+
+![screenshot](./src/assets/images/movies-home.jpg)
+
+This Movie app displays a list of "ocean movies", and the user is able to like any of the movies, and also read more information about each movie once he clicks on the "comments" button. Also, the user is able to leave a comment about any of the movies.
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- Node.js/NPM
+
+
+## Getting Started
+
+Clone the repository from [here](https://github.com/Nuri1977/capstone-js-movies.git)
+
+```
+npm install
+npm run build
+npm start
+```
```

Comment:
- [ ][OPTIONAL] You did great adding commands to install, run and build but I think it would be better to add a command to run the test `npm test` because you have some tests in your project